docs/compute-element/install-htcondor-ce.md

```diff
@@ -17,18 +17,17 @@ Use this page to learn how to install, configure, run, test, and troubleshoot an
 
 !!! note
     If you are installing an HTCondor-CE for use outside of the OSG, consult
-    [the upstream documentation](https://htcondor.github.io/htcondor-ce/v5/installation/htcondor-ce/) instead.
+    [the upstream documentation](https://htcondor.com/htcondor-ce/) instead.
 
 Before Starting
 ---------------
 
 Before starting the installation process, consider the following points, consulting the upstream references as needed
-([HTCondor-CE 5](https://htcondor.github.io/htcondor-ce/v5/reference/),
-[HTCondor-CE 4](https://htcondor.github.io/htcondor-ce/v4/reference/)):
+([HTCondor-CE 5](https://htcondor.com/htcondor-ce/v5/reference/)):
 
 -   **User IDs:** If they do not exist already, the installation will create the Linux users `condor` (UID 4716) and
-    `gratia` (UID 42401)
-    You will also need to create Unix accounts for each VO that you wish to support.
+    `gratia`
```

Comment:
what's the UID for gratia? Unless it's not fixed?

Reply:
Doesn't appear to be fixed

```
%pre common
getent group gratia >/dev/null || groupadd -r gratia
getent passwd gratia >/dev/null || \
       useradd -r -g gratia -c "gratia runtime user" \
       -s /sbin/nologin -d /etc/gratia gratia
```